yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,130 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the keyword value with the given value. Updates the
+          keyword on hardware and also updates on the cache, if cache
+          information  is present in the config JSON.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Inventory/hardware path of the FRU whose keyword needs to be
+                updated.
+          - name: data
+            type:
+                variant[struct[string,string,array[byte]],
+                struct[string,array[byte]]]
+            description: >
+                Contains data required to write. If it is of IPZ type VPD the
+                data should be passed in tuple[record, keyword, value] format
+                and if it is of keyword type VPD the data will be passed in
+                tuple[keyword, data] format, etc. The format of the data will be
+                depend on the type of VPD.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+
+    - name: ReadKeyword
+      description: >
+          Method to read VPD keyword value. Keyword can be read from hardware or
+          from cache.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Inventory/hardware Path of the FRU whose keyword needs to be
+                read.
+          - name: data
+            type: variant[struct[string,string],struct[string]]
+            description: >
+                Contains data requied to read. If it is of IPZ type VPD the data
```

Comment:
keep the brief generic , and add that IPZ and keyword type of VPD support is only added currently and more can be done and also the variant used for them